src/java/picard/illumina/CollectIlluminaLaneMetrics.java

```diff
@@ -49,13 +49,29 @@
  */
 
 @CommandLineProgramProperties(
-        usage = CollectIlluminaLaneMetrics.USAGE,
-        usageShort = CollectIlluminaLaneMetrics.USAGE,
+        usage = CollectIlluminaLaneMetrics.USAGE_SUMMARY + CollectIlluminaLaneMetrics.USAGE_DETAILS,
+        usageShort = CollectIlluminaLaneMetrics.USAGE_SUMMARY,
         programGroup = Illumina.class
 )
 public class CollectIlluminaLaneMetrics extends CommandLineProgram {
-    static final String USAGE = "Collects Illumina lane metrics for the given basecalling analysis directory";
-
+    static final String USAGE_SUMMARY = "Collects Illumina lane metrics for the given BaseCalling analysis directory.  ";
+    static final String USAGE_DETAILS = "This tool produces quality control metrics on cluster density for each lane of an Illumina flowcell." +
+            "  This tool works via distributing Illumina TileMetrics data into directories containing lane- and phasing-level metrics.  " +
+            "In this context, phasing refers to the fraction of molecules that fall behind or jump ahead (prephasing) during a read cycle." +
+            "" +
+            "<h4>Usage example:</h4>" +
+            "<pre>" +
+            "java -jar picard.jar CollectIlluminaLaneMetrics \\<br />" +
+            "      RUN_DIR=130318_SL-HBB_0226_BFCC1WYMACXX \\<br />" +
+            "      OUTPUT_DIRECTORY=Lane_metrics.txt \\<br />" +
+            "      OUTPUT_PREFIX=CollectIlluminaLaneMetrics.txt \\<br />" +
```

Comment:
Ah, no that's not right. Better way to find the right info is to look at the IlluminaLaneMetricsCollectorTest.java file in IntelliJ. What you want is what is specified as the test variables in e.g.

    final static File TEST_DIRECTORY = new File("testdata/picard/illumina/IlluminaLaneMetricsCollectorTest");
    final static File TILE_RUN_DIRECTORY = new File(TEST_DIRECTORY, "tileRuns");

and

        clp.OUTPUT_DIRECTORY = IOUtil.createTempDir("illuminaLaneMetricsCollectorTest", null);
        clp.RUN_DIRECTORY = new File(TEST_DIRECTORY, testRun);
        clp.OUTPUT_PREFIX = "test";
        clp.READ_STRUCTURE = new ReadStructure("101T8B101T");

So you see you can use a simpler name for RUN_DIR; that OUTPUT_DIRECTORY should be a directory name and not a file name, and OUTPUT_PREFIX should be a proper prefix and also not a file name. The latter two are especially important because it would make no sense for a directory name and a prefix to have file extensions.